src/app.js

```diff
@@ -1 +1,7 @@
 'use strict';
+
+const minimist = require('minimist');
+
+const params = minimist(process.argv.slice(2));
```

Comment:
You could use object destructuring for getting values that you need

Reply:
yes, but if I may use minimist, why not? or you think I can't destructure it?))